lib/particular-calendars/france.coutances.ts

```diff
@@ -0,0 +1,244 @@
+import { PatronTitle } from '../constants/martyrology-metadata';
+import { Precedences } from '../constants/precedences';
+import { CalendarDef } from '../models/calendar-def';
+import { Inputs } from '../types/calendar-def';
+
+import { France } from './france';
+
+// src:
+// - mr_fr_1982_ed2_coutances
+// - https://en.wikipedia.org/wiki/Roman_Catholic_Diocese_of_Coutances
+// - https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese
+export class France_Coutances extends CalendarDef {
+  ParentCalendar = France;
+
+  inputs: Inputs = {
+    // src: mr_fr_1982_ed2_coutances
+    our_lady_of_pontmain: {
+      dateDef: { month: 1, date: 17 },
+      precedence: Precedences.ProperMemorial_11b,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    gaud_of_evreux_bishop: {
+      dateDef: { month: 1, date: 30 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    auguste_chapdelaine_priest: {
+      dateDef: { month: 2, date: 28 },
+      precedence: Precedences.ProperMemorial_11b,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    leon_de_carentan_bishop: {
+      dateDef: { month: 3, date: 1 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    placide_eulalie_victoire_jacqueline_viel_virgin: {
+      dateDef: { month: 3, date: 4 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/bienheureuse-marthe-le-bouteiller-vierge
+    marthe_aimee_adele_le_bouteiller_virgin: {
+      dateDef: { month: 3, date: 18 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    paterne_of_avranches_bishop_and_scubilion_abbot: {
+      dateDef: { month: 4, date: 16 },
+      martyrology: ['paterne_of_avranches_bishop', 'scubilion_abbot'],
+      precedence: Precedences.ProperMemorial_11b,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    william_firmatus_abbot: {
+      dateDef: { month: 4, date: 24 },
+      precedence: Precedences.OptionalMemorial_12,
+    },
+
+    // src: mr_fr_1982_ed2_coutances
+    louis_grignion_de_montfort_priest: {},
+
```

Comment:
This memorial is included in the liturgical calendar of this diocese, although nothing has changed compared to the GRC. I still wanted to keep it to indicate that this memorial is also explicitly part of this diocesan calendar (even though none of the properties of this memorial are modified here).

Reply:
> I still wanted to keep it to indicate that this memorial is also explicitly part of this diocesan calendar (even though none of the properties of this memorial are modified here).

While I get that, what is the value of keeping it? :thinking: We have removed such definitions in every other (national) calendar. I think it has no value at all.

Anyway, I don’t actually understand why it is sort of _inscribed_ in the diocesan calendar proper (and yes, I see it in the calendar proper). I though only such celebrations are inscribed which are _added_ on top of the more general calendar (in this case `france.ts`). Usually, not even celebration transfers or ranks changes are listed in the calendar propers (which I think should be), so why here.

---

On second thoughts: I’ve looked into [my sourced list of celebrations inscribed in the GRC](https://gitlab.com/tukusejssirs/romcal_calendars/-/blob/main/1969/general/general.md?ref_type=heads) and now I know why this celebration is listed in the diocesan calendar proper: it was inscribed in the GRC on 20 July 1996 via [_Decretum de celebratione Sancti Ludovici Mariae Grignion de Montfort, presbyteri, in Ca­lendario Romano generali inscribenda_](https://www.vatican.va/archive/aas/documents/AAS-88-1996-ocr.pdf#page=877). :facepalm:

Therefore, I think it can be safely removed, as the celebration is not part of the diocesan calendar proper since 20 July 1996. :wink: